### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.grey.ribbon.label)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2296]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
.card.link[role="link"]:nth-child(6) > .image.ui > .grey.ribbon.label
#### Snippet
`<label class="ui grey right ribbon label">Disabled</label>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.55 (foreground color: #ffffff, background color: #95a5a6, font size: 10.3pt, font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

experiments page: disable light gray contrast, feedback link contrast, gray ribbon doesn't show up against ribbon if image not supplied

![Screen Shot 2019-07-18 at 3 02 59 PM](https://user-images.githubusercontent.com/5615930/61495250-45dab000-a96d-11e9-99a6-f71669ff8445.png)
